### DISCUSSION TITLE
Error creating new Symfony Project

### DISCUSSION THREAD

**Discussion by jordi-gp** [symfony/symfony#48946] [Q&A] [answered]
I'm working on Symfony for my first time and when I use my personal computer I have an issue creating a new project, when I try to create a new project with a skeleton version "^5.4" it gives me the next error:

![imatge](https://user-images.githubusercontent.com/83701435/211765743-fb2953b1-f77f-4179-8f64-8e10bc60aa42.png)

The command that I'm running is
`composer create-project symfony/skeleton:"^5.4" 2023-truiter-symfony`

I'm using a windows10 64bit computer, with PHP 8.1.12 version and version 2.5.1 for composer.

As important information to provide, if I run the command without specifying the version it doesn't has any problem, but when I try to say that must be the version 5 gives it.

**Comment by ahundiak** [comment, marked as answer]
I verified your command works on my linux system.  Normally I would suspect a firewall issue but I think you would have gotten a different sort of error message.  I assume you can install other composer packages.  I guess you could try the Symfony CLI per the docs but I'm guessing it won't help.

**Reply by jordi-gp** [reply]
I tried to install the Symfony CLI for start a new project and the error message doesn't appeared me and the project has been correctly created.

**Comment by stof** [comment]
The `^` character is a special character in the Windows terminal. You need to apply the proper escaping for it to be preserved.

Based on the error message, composer does not receive `^5.4` as version constraint but `5.4` (and the `5.4` version indeed does not exist in the skeleton as the skeleton version is 5.4.99, not 5.4.0)

**Reply by stof** [reply]
IIRC, `composer create-project "symfony/skeleton:^5.4"` (i.e. adding quotes around the whole argument and not just part of it) should work.

**Reply by jordi-gp** [reply]
I tried to use the command adding quotes arround the whole argument but the problem remains beeing the same.
